CONTRIBUTING.md

```diff
@@ -43,7 +43,7 @@ Specifically for code in this repo:
    [Biome](https://biomejs.dev/). Execute the following command to format and lint your code at the root of this project before submitting a pull request:
 
 ```bash
-pnpm check:fix
+pnpm fix
```

Comment:
Did you update package.json?

Reply:
The command was wrong, package.json only contains [fix script](https://github.com/solana-developers/program-examples/blob/45e302abb5213363fef5f4db9defbcf51a5c794a/package.json#L10)